build/validate-bicep.sh

```diff
@@ -5,6 +5,7 @@ then
     exit 1
 fi
 
+WARNING_MSG="WARNING: The following experimental Bicep features have been enabled: Extensibility, Dynamic type loading, Provider registry. Experimental features should be enabled for testing purposes only, as there are no guarantees about the quality or stability of these features. Do not enable these settings for any production usage, or your production environment may be subject to breaking."
```

Comment:
Will look into if this can be disabled so it doesn't print on every build

Reply:
For now if this is the output, we'll ignore it since it's not an error

Reply:
What if this message changes? How will we know that we need to update this?

Reply:
The workflow would start failing. I think the ideal situation would be to suppress the warning message if that's possible

Reply:
Possible we could turn this into a wildcard so we don't have to match the whole string? eg: `WARNING: The following experimental Bicep features *`

I'm wondering about whether the string is going to change and break our build.